src/styleguide/_data/data.json

```diff
@@ -5,6 +5,9 @@
     "medium" : "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
     "long" : "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
   },
+  "preformatted-text": {
+    "ascii-table-flip": "(╯°Д°)╯︵ ┻━┻"
```

Comment:
The variable name is too specific, in case you want to override its value in another page, just use `preformatted-text` would be ok if you think that `performatted-text` would have more than one prop, then use a more generic one like `ascii` ?